khelo/lib/components/action_bottom_sheet.dart

```diff
@@ -53,20 +53,22 @@ class BottomSheetAction extends StatelessWidget {
   final Widget? icon;
   final String title;
   final Widget? child;
+  final bool enabled;
   final void Function()? onTap;
 
   const BottomSheetAction({
     super.key,
     this.icon,
     required this.title,
+    this.enabled = true,
     this.child,
     this.onTap,
   });
 
   @override
   Widget build(BuildContext context) {
     return OnTapScale(
-      enabled: child == null,
+      enabled: enabled,
```

Comment:
have you managed other places where this shouldn't be enabled i.e more option in filter teams ??